envi/archs/ppc/disasm_gen.py

```diff
@@ -0,0 +1,82 @@
+# THIS SHOULD BE COPIED INTO disasm.py FOR DEPENDENCIES SAKE
```

Comment:
that's more instructions than a TODO. disasm_gen.py is generated by mnems.py.  if you look, the data is indeed copied in.  let me know if you'd like me to stop tracking disasm_gen.py to avoid this confusion.